SQL/Archive/2017-11-01-use_consent_to_database.sql

```diff
@@ -0,0 +1,35 @@
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'useConsent', 'Enable if the study uses the loris architecture for consent', 1, 0, 'boolean', ID, 'Use consent', 15 FROM ConfigSettings WHERE Name='study';
+INSERT INTO Config (ConfigID, Value) SELECT ID, 'false' FROM ConfigSettings WHERE Name='useConsent';
```

Comment:
front-end to-do: consent status tab to render only if this is set true. remove use of config.xml